sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/application-offering-change-request.aest.controller.ts

```diff
@@ -0,0 +1,59 @@
+import { Controller, Get, Query } from "@nestjs/common";
+import { ApiTags } from "@nestjs/swagger";
+import { AuthorizedParties } from "../../auth/authorized-parties.enum";
+import { AllowAuthorizedParty, Groups } from "../../auth/decorators";
+import { ClientTypeBaseRoute } from "../../types";
+import { PaginatedResultsAPIOutDTO } from "../models/pagination.dto";
+import BaseController from "../BaseController";
+import {
+  InProgressApplicationOfferingChangesAPIOutDTO,
+  InProgressOfferingChangePaginationOptionsAPIInDTO,
+} from "./models/application-offering-change-request.institutions.dto";
+import { ApplicationOfferingChangeRequestService } from "../../services";
+import { ApplicationOfferingChangeRequestStatus } from "@sims/sims-db";
+import { UserGroups } from "../../auth";
+import { ApplicationOfferingChangeRequestControllerService } from "..";
+
+/**
+ * Application offering change request controller for ministry client.
+ */
+@AllowAuthorizedParty(AuthorizedParties.aest)
+@Groups(UserGroups.AESTUser)
+@Controller("application-offering-change-request")
+@ApiTags(`${ClientTypeBaseRoute.AEST}-application-offering-change-request`)
+export class ApplicationOfferingChangeRequestAESTController extends BaseController {
+  constructor(
+    private readonly applicationOfferingChangeRequestService: ApplicationOfferingChangeRequestService,
+    private readonly applicationOfferingChangeRequestControllerService: ApplicationOfferingChangeRequestControllerService,
+  ) {
+    super();
+  }
+
+  /**
+   * Get all in progress application offering change requests.
+   * @param pagination options to execute the pagination.
+   * @returns list and count of inprogress application offering change requests.
+   */
+  @Get("in-progress")
+  async getAllInProgressApplications(
+    @Query() pagination: InProgressOfferingChangePaginationOptionsAPIInDTO,
+  ): Promise<
+    PaginatedResultsAPIOutDTO<InProgressApplicationOfferingChangesAPIOutDTO>
+  > {
+    const offeringChange =
+      await this.applicationOfferingChangeRequestService.getSummaryByStatus(
```

Comment:
In the service, I see just one level of sort happening and while sorting by status, is this order of sort happen naturally ? 
![image](https://github.com/bcgov/SIMS/assets/54600590/a90c3ca2-664a-4316-a84e-25de6b3bbedd)

Reply:
While sorting by status `DESC` it comes first "with SABC" and then "with Student". I just fixed that to bring by Date submitted `ASC` when sorting by status. Thanks.